packages/oas-utils/src/transforms/import-spec.ts

```diff
@@ -215,21 +217,30 @@ export async function importSpecToWorkspace(
           flow.type = key
 
           // Prefill values from authorization config - old deprecated config
+          // @ts-expect-error - deprecated
```

Comment:
@hanspagel how long should we support the old config? we can do a minor release and just remove it soon I thnk